util/omitUndefinedFields.ts

```diff
@@ -13,7 +13,7 @@ export default function omitUndefinedFields<T>(object: T): T {
     }
     return Object.fromEntries(
         Object.entries(object)
-            .filter(([, value]) => value !== undefined)
+            .filter(([, value]) => value !== null)
             .map(([key, value]) => {
```

Comment:
Did you encounter an issue with `null` in `omitUndefinedFields`? Here, we need to filter `undefined` here because Next.js has an issue with it, but it doesn't have an issue with `null`. I believe there's a bug in `omitUndefinedFields` because `typeof null` evaluates to `'object'` but the above check doesn't handle that. I think if we fix that bug, then we can keep this `filter` line unchanged.

Reply:
I think adding `|| value === null` here (above) might fix the null bug, returning early if we have a null value.
```ts
    if (typeof object !== 'object' || value === null) {
        return object
    }
```

Reply:
Sorry, I just realized my snippet is wrong, the variable is named `object`, not `value`

Reply:
All good! I flipped the logic, which I thought may make sense. What do you think?

```
if (object && typeof object === 'object')
```
If it's not `null` and is an object then do the thing. Otherwise return the object.